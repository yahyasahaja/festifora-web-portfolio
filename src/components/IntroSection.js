import LinkedinSvg from '../assets/images/linkedin-brands.svg'
import GithubSvg from '../assets/images/github-brands.svg';
import TwitterSvg from '../assets/images/twitter-brands.svg';

function App() {
  return (
    <section data-testid="intro" className="intro [ min-h-screen ] [ flex justify-center items-center flex-col ] ">
      <h1 className="name text-5xl">Nama Saya Budi</h1>
      <div className="tag-line text-2xl mt-5">
        Budi Pemain Bola
      </div>
      <div className="links flex mt-5">
        <a href="https://linkedin.com/in/budi" target="_blank" rel="noreferrer">
          <img className="w-10 h-10 m-3" src={LinkedinSvg} alt="linkedin" />
        </a>
        <a href="https://github.com/budi" target="_blank" rel="noreferrer">
          <img className="w-10 h-10 m-3" src={GithubSvg} alt="linkedin" />
        </a>
        <a href="https://twitter.com/budi" target="_blank" rel="noreferrer">
          <img className="w-10 h-10 m-3" src={TwitterSvg} alt="linkedin" />
        </a>
      </div>
    </section>
  );
}

export default App;
