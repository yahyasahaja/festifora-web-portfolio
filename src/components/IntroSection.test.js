import { render, screen } from '@testing-library/react';
import IntroSection from './IntroSection';

test('renders the element correctly', () => {
  render(<IntroSection />);
  const wrapper = screen.getByTestId('intro');
  expect(wrapper).toMatchSnapshot();
});
