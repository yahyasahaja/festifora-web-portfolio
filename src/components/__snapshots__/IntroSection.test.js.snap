// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the element correctly 1`] = `
<section
  class="intro [ min-h-screen ] [ flex justify-center items-center flex-col ] "
  data-testid="intro"
>
  <h1
    class="name text-5xl"
  >
    Nama Saya Budi
  </h1>
  <div
    class="tag-line text-2xl mt-5"
  >
    Budi Pemain Bola
  </div>
  <div
    class="links flex mt-5"
  >
    <a
      href="https://linkedin.com/in/budi"
      rel="noreferrer"
      target="_blank"
    >
      <img
        alt="linkedin"
        class="w-10 h-10 m-3"
        src="linkedin-brands.svg"
      />
    </a>
    <a
      href="https://github.com/budi"
      rel="noreferrer"
      target="_blank"
    >
      <img
        alt="linkedin"
        class="w-10 h-10 m-3"
        src="github-brands.svg"
      />
    </a>
    <a
      href="https://twitter.com/budi"
      rel="noreferrer"
      target="_blank"
    >
      <img
        alt="linkedin"
        class="w-10 h-10 m-3"
        src="twitter-brands.svg"
      />
    </a>
  </div>
</section>
`;
